### DISCUSSION TITLE
Smart Contract's methods not exported

### DISCUSSION THREAD

**Discussion by Daud94** [ethereum/web3.js#5594] [Q&A]
Functions in my smart contract are not available to be called after deployment to Ganache. What could be the cause?

**Comment by dawgbytes** [comment]
Could web3.js be used instead?  or remix solidarity to eth or bsc?

**Reply by TatyOko28** [reply]
Yes, you can use Web3.js instead of Remix.
 You can deploy to Ethereum or Binance Smart Chain (BSC) by switching networks.

**Comment by TatyOko28** [comment]
If functions in your smart contract are not available after deployment, the possible reasons include:
 The contract was not deployed correctly (wrong ABI or address).
 The methods are not public or external in Solidity.
 The wrong contract instance is being used in Web3.js.
 The contract is not properly linked to Web3.js (wrong provider, ABI, or contract address).
